docs/modules/ROOT/pages/serverless-plugin.adoc

```diff
@@ -0,0 +1,110 @@
+= Defender Serverless Plugin
+
+Defender Serverless is a Serverless Framework plugin for automated resource management on Defender.
+
+== Installation
+
+You can initialise your Serverless project directly using our pre-configured template:
+
+```
+sls install --url https://github.com/OpenZeppelin/defender-serverless/tree/main/template -n my-service
+```
+
+Alternatively, you can install it directly into an existing project with:
+
+`yarn install defender-serverless`
+
+== Setup
+
+This plugin allows you to define Autotasks, Sentinels, Notifications, Relayers, Contracts, Policies and Secrets declaratively from a `serverless.yml` and provision them via the CLI using `serverless deploy`. An example template below with an autotask, a relayer, a policy and a single relayer API key defined:
+
+```yaml
+service: defender-serverless-template
+configValidationMode: error
+frameworkVersion: '3'
+
+provider:
+  name: defender
+  stage: ${opt:stage, 'dev'}
+  stackName: 'mystack'
+  ssot: false
+
+defender:
+  key: '${env:TEAM_API_KEY}'
+  secret: '${env:TEAM_API_SECRET}'
+
+functions:
+  autotask-example-1:
+    name: 'Hello world from serverless'
+    path: './autotasks/hello-world'
+    relayer: ${self:resources.Resources.relayers.relayer-1}
+    trigger:
+      type: 'schedule'
+      frequency: 1500
+    paused: false
+
+resources:
+  Resources:
+    policies:
+      policy-1:
+        gas-price-cap: 1000
+        whitelist-receivers:
+          - '0x0f06aB75c7DD497981b75CD82F6566e3a5CAd8f2'
+        eip1559-pricing: true
+
+    relayers:
+      relayer-1:
+        name: 'Test Relayer 1'
+        network: 'rinkeby'
+        min-balance: 1000
+        policy: ${self:resources.Resources.policies.policy-1}
+        api-keys:
+          - key1
+
+plugins:
+  - defender-serverless
+```
+
+This requires setting the `key` and `secret` under the `defender` property of the YAML file. We recommend using environment variables or a secure (gitignored) configuration file to retrieve these values. Modify the `serverless.yml` accordingly.
+
+Ensure the Defender Team API Keys are setup with all appropriate API capabilities.
+
+The `stackName` (e.g. mystack) is combined with the resource key (e.g. relayer-1) to uniquely identify each resource. This identifier is called the `stackResourceId` (e.g. mystack.relayer-1) and allows you to manage multiple deployments within the same Defender team.
+
+NOTE: A caveat is when `ssot` (single source of truth) is enabled, all resources on your account that do not belong to the current stack are removed upon deployment.
```

Comment:
This note is a bit hard to grok for me, maybe a little section dedicated to "ssot mode" (and linking it from here) would help. Especially considering it sounds like it could have some impactful consequences